examples/chart/teleport-kube-agent/.lint/updater.yaml

```diff
@@ -0,0 +1,6 @@
+proxyAddr: proxy.example.com:3080
+roles: ""
+updater:
+  enabled: true
+  versionServer: https://my-custom-version-server/v1
+  releaseChannel: cloud/preview
```

Comment:
Can we have the default value here? or something which says `custom`?
Otherwise, may lead to people assuming there's an official `cloud/preview` channel (even if the versionServer is clearly _custom_)